src/view/com/lightbox/ImageViewing/index.tsx

```diff
@@ -277,13 +421,77 @@ const styles = StyleSheet.create({
   },
 })
 
-const EnhancedImageViewing = (props: Props) => (
-  <ImageViewing key={props.initialImageIndex} {...props} />
-)
+function ImageViewingRoot({
+  lightbox: nextLightbox,
+  onRequestClose,
+  onPressSave,
+  onPressShare,
+}: {
+  lightbox: Lightbox | null
+  onRequestClose: () => void
+  onPressSave: (uri: string) => void
+  onPressShare: (uri: string) => void
+}) {
+  const [activeLightbox, setActiveLightbox] = useState(nextLightbox)
```

Comment:
note! forking props into state. this is intentional because we want to treat new `lightbox` prop values as sort of "requests" — and trigger animations in response